CHANGELOG.md

```diff
@@ -13,6 +13,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `--color` option to control when colored output is used
 
+#### Changed
+- `start_warp` and `stop_warp` now take `CheatTarget` as the first argument instead of `ContractAddress`. Read more [here](./docs/src/appendix/cheatcodes/start_warp.md). 
+
```

Comment:
we should also mention `start_prank` here

Reply:
I added an identical bullet point for `start_prank`, but once all the cheats use the new interface we should probably just consolidate them into 1 bullet point.